Frameworks/UIKit/UIViewController.mm

```diff
@@ -2245,7 +2253,6 @@ - (void)dealloc {
 
     priv->_childViewControllers = nil;
     priv->_searchDisplayController = nil;
-    priv->_storyboard = nil;
```

Comment:
I see that the UIViewController.mm `setStoryboard:` method isn't exposed publicly so technically there's no crash in the current code (but the ivar should still be changed to a strong id)